base-notebook/Dockerfile

```diff
@@ -51,10 +51,9 @@ RUN apt-get -q update \
     sudo \
     locales \
     fonts-liberation \
-    run-one \
- && apt-get clean && rm -rf /var/lib/apt/lists/*
-
-RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
+    run-one && \
```

Comment:
Please, fix the style for `&&` several lines above as well.

Reply:
Hello,

I do not understand what need to be fixed here regarding the style, sorry.

```dockerfile
fonts-liberation \
run-one && \
apt-get clean && rm -rf /var/lib/apt/lists/* && \
echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
locale-gen
```

Reply:
Several lines above the line start from `&&`like this: `&& apt-get install -yq --no-install-recommends \`

Reply:
Ok got it, I though the comment was related to my change, but Ok I've seen it.

Reply:
Thanks :)
Sorry for being not 100% clear.